src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -100,18 +104,19 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         // No validation
-        internal CSharpParseOptions(
+        private CSharpParseOptions(
```

Comment:
there is actually 2 references to this method

Status	Code	File	Line	Column	Project
	CSharpParseOptions.CSharpParseOptions(LanguageVersion, DocumentationMode, SourceCodeKind, ImmutableArray<string>, ImmutableDictionary<string, string>)				
	: this(languageVersion,	C:\dd\roslyn\src\Compilers\CSharp\Portable\CSharpParseOptions.cs	51	15	CSharpCodeAnalysis
	private CSharpParseOptions(CSharpParseOptions other) : this(	C:\dd\roslyn\src\Compilers\CSharp\Portable\CSharpParseOptions.cs	97	64	CSharpCodeAnalysis